### ISSUE TITLE
Word count for a selected text

### ISSUE THREAD

**Issue by zyrikby** [James-Yu/LaTeX-Workshop#224]
I've tested the word count feature and it works well for the whole document. However, sometimes you need to count the words only in a current tex file (e.g., if you have a multi-chapter book you need to count words in a current chapter) or even for a selected text. Is it possible to implement this feature?

**Comment by James-Yu** [comment]
Not quite possible for selected text as TeXCount seems not providing this option. Correct me if I'm wrong.

Counter for a file is possible. Though I need to think of ways to present the option elegantly, or at least not ugly.

**Comment by James-Yu** [comment]
Implemented and will be updated on marketplace shortly.

Now the logic is like this. If the current active document is the root file, or is not a `tex` file, then the counter returns total words and floats. Otherwise count the opened file.

Hope this logic is intuitive and sufficient.

**Comment by Thinking2018** [comment]
Hi, Yu,

I cannot use your word count function in the extension of latex-workshop.

I have installed active perl and texcount.pl, and have set the texcount.path to the file. But when I click "Count word in the Latex project", nothing happens there.

I have searched all the documents, there is no such situation like this. I really don't know what's happens there. It works when I type the command in the terminal.
